stories/10-Switch.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import {storiesOf} from "@storybook/react";
+
+import Switch from "../src/switch/Switch";
+import StateProvider from "./utils/StateProvider";
+import StoryFragment from "./utils/StoryFragment";
+
+storiesOf("Switch", module).add("Switch States", () => {
+  const initialState = {
+    isSwitchOn: true,
+    isDisabled: false
+  };
+
+  const darkTheme = (
+    <style>{`
+    .sb-show-main {
+      background-color: gray;
+    }
```

Comment:
```suggestion
      background-color: gray;
      color: white;
    }
```
Not a big deal, but i think it is a good touch 😄